tox.ini

```diff
@@ -7,3 +7,11 @@ skipsdist = true
 deps = -r scripts/nb-tester/requirements.txt
 setenv = PYDEVD_DISABLE_FILE_VALIDATION=1
 commands = python scripts/nb-tester/test-notebook.py {posargs}
+
+[testenv:lint]
+deps = -r scripts/nb-tester/requirements.txt
```

Comment:
I don't think you need to keep specifying this since it's in the default `[testenv]`

Reply:
Fixed, thanks!